mkt/site/tasks.py

```diff
@@ -40,15 +42,16 @@ def send_email(recipient, subject, message, real_email, from_email=None,
 
 
 @task
-def set_modified_on_object(obj, **kw):
+def set_modified_on_object(app_label, model_name, pk, **kw):
```

Comment:
No, it's fine. We should have a bug about it, it's worth fixing (would make things simpler and reduce the amount of tasks in the queue)